client/src/LoadClientDetails/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from "react";
+import LoadClientContext from "../LoadClientContext";
+
+import "./styles.css"
+ 
+const LoadClientDetails = props => {
+  const { loadclients, currClientDetails, handleClose } = useContext(LoadClientContext);
+
```

Comment:
To cut back on code, can create a variable here and reference its properties in the JSX
const details = loadClients[currentClientDetails]

```<h1>{details.name}</h1>```